src/docfx/config/Config.cs

```diff
@@ -23,9 +25,14 @@ internal sealed class Config
         public readonly string Name = string.Empty;
 
         /// <summary>
-        /// Gets the contents that are managed by this docset.
+        /// Gets the file glob patterns included by the docset.
+        /// </summary>
+        public readonly string[] Files = s_defaultInclude;
```

Comment:
how about
```
content:
  include:
  exclude:
```

Reply:
or use `!` to represent exclude, only `files`
```
files:
  - a.md
  - !b.md
```

Reply:
The final schema is `files` and `exclude`.